content/docs/tutorials/basic/node.md

```diff
@@ -203,8 +200,7 @@ There are two parts to making our `RouteGuide` service do its job:
   service responses.
 
 You can find our example `RouteGuide` server in
-[examples/node/dynamic&#95;codegen/route&#95;guide/route&#95;guide&#95;server.js](https://github.com/grpc/grpc/blob/
-{{< param grpc_release_tag >}}/examples/node/dynamic_codegen/route_guide/route_guide_server.js).
+[examples/node/dynamic&#95;codegen/route&#95;guide/route&#95;guide&#95;server.js](https://github.com/grpc/grpc/blob/{{< param grpc_release_tag >}}/examples/node/dynamic_codegen/route_guide/route_guide_server.js).
```

Comment:
Use plain slashes here instead of the HTML character (and elsewhere in this doc as well)

Reply:
You mean plain underscores, right?

(We can probably handle that kind of cleanup -- in the rest of the doc -- separately. But feel free to do it if you feel inclined to Mario).

Reply:
Followup issue: #162